server/backend/database/memory/database.go

```diff
@@ -666,6 +667,47 @@ func (d *DB) CreateChangeInfos(
 	return nil
 }
 
+// DeleteOldChangeInfos delete old change infos before checpoint to save storage
+func (d *DB) DeleteOldChangeInfos(
+	ctx context.Context,
+	docInfo *database.DocInfo,
+) error {
+	txn := d.db.Txn(true)
+	defer txn.Abort()
+
+	// 01. Find the smallest serverSeq recorded in colSyncedSeqs.
+	// This is to avoid the error that the change could not be found when doing UpdateAndFindMinSyncedTicket.
+	it, err := txn.Get(tblSyncedSeqs, "id")
+	if err != nil {
+		return err
+	}
+
+	var minSyncedServerSeq int64 = math.MaxInt64
```

Comment:
```suggestion
	minSyncedServerSeq := math.MaxInt64
```

https://github.com/uber-go/guide/blob/master/style.md#local-variable-declarations

Reply:
Thank you so much for commenting.
I should have checked the guide more carefully, but I think I missed it..
We will update it.

Reply:
https://github.com/chromato99/yorkie/blob/7f04224d979743882d885bb42dce4447af5d3fef/server/backend/database/memory/database.go#L685

Go recognized MaxInt64 as an int instead of an int64, so I got an error in the comparison statement below.
Would this method be okay?